internal/gitserver/search/match_tree.go

```diff
@@ -215,6 +217,14 @@ func (dmf *DiffModifiesFile) Match(lc *LazyCommit) (bool, MatchedCommit, error)
 	}, nil
 }
 
+type Constant struct {
+	Value bool
+}
+
+func (c *Constant) Match(_ *LazyCommit) (bool, MatchedCommit, error) {
```

Comment:
```suggestion
func (c *Constant) Match(*LazyCommit) (bool, MatchedCommit, error) {
```

I believe this is fine?

Reply:
Yep, definitely more conventional